backend/YouTubeV2.Application/Model/Role.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Identity;
+
+namespace YouTubeV2.Application.Model
+{
+    public class Role : IdentityRole
+    {
+        public static readonly string User = "USER";
+        public static readonly string Admin = "ADMIN";
+
+        public virtual ICollection<UserRole> UsersRoles { get; private set; }
+
+        public Role(string roleName) : base(roleName)
+        {
+            NormalizedName = roleName.ToUpper();
```

Comment:
is it needed?

Reply:
Yes. WIthout it NormalizeNAme is null and we get "Role User does not exist" error